src/main/webapp/portal/main/partials/header.html

```diff
@@ -8,7 +8,7 @@
                  <span class="icon-bar"></span>
              </button>
              <div class="header-mobile">
-               <h3><a class="header-text" href="./" id="myuw-header"><img ng-src="{{::headerCtrl.crest}}" class="hidden-xs" alt="{{::headerCtrl.crestalt}}"><span>{{NAMES.title}}</span><span class="beta-logo">{{NAMES.sublogo}}</span></a></h3>
+               <h3><a aria-label="My U W" class="header-text" href="./" id="myuw-header"><img ng-src="{{::headerCtrl.crest}}" class="hidden-xs" alt="{{::headerCtrl.crestalt}}"><span>{{NAMES.title}}</span><span class="beta-logo">{{NAMES.sublogo}}</span></a></h3>
```

Comment:
This looks like the right text to be adding, but it should probably be externalized into `NAMES.title-aria-label` or so, so as to be externalized parallel to `NAMES.title`.